content/blog/2018/beyondtellerand.md

```diff
@@ -87,10 +87,6 @@ favorites:
   caption='Video from Beyond Tellerand'
 ) }}
 
-{{ contact.subscribe(
-  interests=['oddnews', 'beyondtellerand19']
-) }}
-
```

Comment:
@mirisuzanne Do we want to just remove this everywhere? Looks like it's only being used on one post (vueconf).

Reply:
@mirisuzanne LGTM pending your thoughts on ☝️